.github/workflows/rust.yml

```diff
@@ -123,6 +123,10 @@ jobs:
     - name: Run tests
       run: cargo nextest run
 
+    - name: Run documentation tests
+      # Nextest doesn't have the `--doc` flag, so we are forced to use `cargo test` here.
+      run: cargo test --doc
+
```

Comment:
can you instead change the recipe in xtask for test [here](https://github.com/tursodatabase/libsql/blob/main/xtask/src/main.rs#L75), and then have the CI run `cargo xtask test`?

Reply:
Done